kinbot/mess.py

```diff
@@ -1,27 +1,44 @@
 from __future__ import division
+from __future__ import print_function
+from shutil import copyfile
+import sys
+import json
+import logging
 import os
+import datetime
 import subprocess
 import time
 import pkg_resources
-import logging
+import math
+import random
+import numpy as np
+
 from kinbot import constants
 from kinbot import frequencies
-from kinbot import cheminfo
+from kinbot import license_message
+from kinbot.parameters import Parameters
+
 
 class MESS:
     """
-    Class that reads and write all MESS files and runs MESS
+    Class that read and writes MESS files for UQ analysis
+    Code is the same as mess.py exceppt that it loops to create 'n' files
+    and it randomizes the following parameters within the alloted UQ tolerance
+       1. Stationary point energy (E+ZPE, +/- 0.5 kcal/mol)
+       2. Barrier (E+ZPE, +/- 1.0 kcal/mol)
+       3. Frequencies (cm-1 +/-20%)
+    Ranges were chosen/based on the following paper:  Goldsmith, C. F. PCI, 2013, 177-185
+    'n' = number of input files generated, default = X (will be implemented once code works)
     """
+
     def __init__(self, par, species):
         self.par = par
         self.species = species
         self.well_names = {}
         self.bimolec_names = {}
         self.fragment_names = {}
         self.ts_names = {}
-        self.termolec_names = {}
-        self.barrierless_names = {}
```

Comment:
You deleted this and large chunks related to termolecular and barrierless reactions. What is it about?

Reply:
I added this information back in, I think the mess file I was working off of was from prior to the additions of barrierless and termolecular blocks to the mess.py file.